acct-user/miniflux/miniflux-0.ebuild

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit acct-user
+
+DESCRIPTION="User for www-apps/miniflux"
+ACCT_USER_ID=404
+ACCT_USER_HOME="/var/empty"
```

Comment:
Why not simply not define this variable?

Reply:
Ah, good point. I didn't think to try that so it would default to `/dev/null`.